django_select2/forms.py

```diff
@@ -108,6 +108,21 @@ def _get_media(self):
 
     media = property(_get_media)
 
+    def label_from_instance(self, obj):
+        """
+        Label representation from instance.
+
+        Can be overridden to change the representation of each choice.
```

Comment:
Don't forget to describe behavior of the default implementation. Make sure users get, that instances need have a string representation (to implement `__str__`).